src/schemaDesigner/schemaDesignerWebviewController.ts

```diff
@@ -75,11 +80,23 @@ export class SchemaDesignerWebviewController extends ReactWebviewPanelController
         });
 
         this.registerRequestHandler("initializeSchemaDesigner", async () => {
-            const sessionResponse = await this.schemaDesignerService.createSession({
-                connectionString: this.connectionString,
-                accessToken: this.accessToken,
-                databaseName: this.databaseName,
-            });
+            let sessionResponse: SchemaDesigner.CreateSessionResponse;
```

Comment:
It seems we are only storing state after we perform certain sts based actions? 

I guess we should create a new request handler just for syncing state. Since this won't capture some user actions like moving tables/ editing tables.